test/unit/host_discovered_test.rb

```diff
@@ -26,7 +26,6 @@ class HostDiscoveredTest < ActiveSupport::TestCase
   end
 
   test "should setup subnet" do
-    raw = parse_json_fixture('/facts.json')
```

Comment:
I removed many of these instances. They're worthless as far as I can tell. Tests pass without them, moreover the `raw` variables are not even referenced later.